### DISCUSSION TITLE
Welcome to core Discussions!

### DISCUSSION THREAD

**Discussion by ocombe** [ocombe/ng2-translate#1271] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by carlostipton1319** [comment]
Yes, that's correct. In Angular, lazy-loaded modules use a separate [contractors](https://alraihanrental.com/) injector, allowing you to configure them independently from the rest of your application

**Comment by digitalsohailseo** [comment]
Indeed, that's accurate. In Angular, lazy-loaded modules use a separate constructor's injector, granting you the ability to configure them independently from the rest of your application with the precision of an [SEO expert in Pakistan](https://www.sohailzafar.me/).
